common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1289,6 +1295,15 @@ def _sorting_dates(self):
         return announcement, start, now
 
     @lazy
+    def grading(self):
```

Comment:
> Please add comment why we go this way.

I don't think that it should be a part of the docstring, we can add this info in PR to edx.